File: src/python/fibo.py
firstElement = 0
secondElement = 1
count = 0
result = firstElement + secondElement
fiboNumber = input("input fibo number:")

while count < fiboNumber:
  thirdElement = firstElement + secondElement
  result += thirdElement
  firstElement = secondElement
  secondElement = thirdElement
  count += 1
print("fibo result is {} and calculation result is {}".format(thirdElement, result))

File: src/javacode/algorithm/Fibo.java
package javacode.algorithm;

public class Fibo {
  
  public static void main(String[] args) {
    System.out.println("start fibo calculation in java");
    int count = 0,firstEle = 0,secondEle = 1,currentEle = 0,result = 0;
    result = firstEle + secondEle;
    int arg = Integer.valueOf(args[0]).intValue();
    for (int i = 0; i < arg; i++) {
      currentEle = firstEle + secondEle;
      result += currentEle;
      firstEle = secondEle;
      secondEle = currentEle;
    }
    System.out.println("fibo number is:" + currentEle + " and calculation result is:" + result);
  }
}
